### PR TITLE
Add Jira ports of pretriage and posttriage

### PR DESCRIPTION
**Pull request by pierreprinetti**
Post-triage is essentially a placeholder as it only checks for Priority. We will have to add checks for Severity and `qe_test_coverage` when we know where those fields end up in Jira.

Corresponding Prow PR: https://github.com/openshift/release/pull/30023

Implements [OSASINFRA-2912](https://issues.redhat.com//browse/OSASINFRA-2912)
Implements [OSASINFRA-2913](https://issues.redhat.com//browse/OSASINFRA-2913)

### CODE REVIEW

**Review by stephenfin — COMMENTED** [review]
Couple of questions inline but nothing that would block this.

/lgtm

File: README.md [hunk 1]
```diff
@@ -2,50 +2,62 @@
 
 ## pretriage
 
-Usage: `./pretriage.py`
+Usage:
+* for Bugzilla: `./pretriage.py`
+* for Jira: `go build ./cmd/pretriage && ./pretriage`
```
[the remark this answers is not in the document]
Comment (pierreprinetti) [reply README.md:7]: I have no plans of unifying the two tools, as new bugs will be Jira-only soon. We can remove the Python Bugzilla tool as soon as we are done with the Bugzilla bug backlog

File: README.md [hunk 2]
```diff
@@ -2,50 +2,62 @@
...
+* `SLACK_HOOK`: a Slack hook URL
 * `TEAM_MEMBERS` is a JSON object in the form:
 
 ```json
 [
   {
     "slack_id": "UG65473AM",
-    "bz_id": "user1@example.com"
+    "bz_id": "user1@example.com",
+    "components": ["component1"],
```
[the remark this answers is not in the document]
Comment (pierreprinetti) [reply README.md:23]: The easiest way forward I could think of, is to just leave bugzilla fields there until we clear our Bugzilla bug backlog

File: README.md [hunk 3]
```diff
@@ -2,50 +2,62 @@
...
 
 Finds untriaged, unassigned Shiftstack bugs and assigns them to a team member.
 
 Required environment variables:
 
-* `BUGZILLA_API_KEY`: a [Bugzilla API key](https://bugzilla.redhat.com/userprefs.cgi?tab=apikey). 
-* `SLACK_HOOK`: a Slack hook URL.
+* `BUGZILLA_API_KEY`: a [Bugzilla API key](https://bugzilla.redhat.com/userprefs.cgi?tab=apikey)
+* `JIRA_TOKEN`: a [Jira API token](https://issues.redhat.com/secure/ViewProfile.jspa?selectedTab=com.atlassian.pats.pats-plugin:jira-user-personal-access-tokens) of an account that can access the OCPBUGS project
+* `SLACK_HOOK`: a Slack hook URL
```
[the remark this answers is not in the document]
Comment (pierreprinetti) [reply README.md:15]: :+1: https://github.com/shiftstack/bugwatcher/pull/34

**Pull request merged**